n3fit/src/n3fit/backends/keras_backend/base_layers.py

```diff
@@ -30,14 +31,18 @@ def square_activation(x):
     """ Squares the input """
     return x*x
 
+def modified_tanh(x):
+    return (nn.relu(x)+nn.relu(-x))*nn.tanh(x)
```

Comment:
Please add some docstring.
Also, why `relu(x) + relu(-x)` instead of `abs(x)`? Wouldn't it be the same?

Reply:
I think it had something to do with how the gradient was defined in x=0, but let me see if that is correct.

Reply:
I thought I saw somewhere that the gradient of abs(x) was not well defined at x=0 while that of relu just goes to 0. Not sure why I thought that: I just explicitly tested and the gradient of abs(x) at x=0 is also 0.